docs/content/en/tasks/backup-restore/_index.md

```diff
@@ -115,32 +115,27 @@ To perform a backup of a Cassandra datacenter, create the following custom resou
 
 ```yaml
 apiVersion: medusa.k8ssandra.io/v1alpha1
-kind: CassandraBackup
+kind: MedusaBackupJob
 metadata:
   name: medusa-backup1
 spec:
   cassandraDatacenter: dc1
-  name: medusa-backup1
 ```
 
-The `metadata.name` value can match the `spec.name` value for convenience, but it is not mandatory. The latter will be used to identify the backup in the storage backend, the former being the name of the `CassandraBackup` custom resource in Kubernetes.
-
 ### Checking Backup Completion
 
-K8ssandra Operator will detect the `CassandraBackup` object creation and trigger a backup asynchronously.
+K8ssandra Operator will detect the `MedusaBackupJob` object creation and trigger a backup asynchronously.
 
-To monitor the backup completion, check if the `finishTime` value isn't empty in the CassandraBackup object status. Example:
+To monitor the backup completion, check if the `finishTime` value isn't empty in the `MedusaBackupJob` object status. Example:
```

Comment:
I think it is more clear to state something in the affirmative when possible. What do you think about about,

>To monitor the backup completion, check if the `finishTime` is set in the `MedusaBackupJob` object status. Example: